web/profiles/joinup/config/install/core.entity_view_mode.node.rss.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
RSS? I don't think I've seen this in the functional specs yet. It's also here because we copied the standard profile. Let's remove this display mode for now, we can still add it back later if needed in a dedicated ticket.